tests/core/test_registry.py

```diff
@@ -141,3 +142,8 @@ def test_unable_to_activate_nonexistent_aliased_registry(config):
     add_registry(URI_1, "mine", config)
     with pytest.raises(InstallError):
         activate_registry("other", config)
+
+
+def test_explore_uri_invalid_raises_error(config):
```

Comment:
This looks like a great place to use [`@pytest.mark.parametrize`](https://docs.pytest.org/en/latest/parametrize.html) - we're big fans of  this decorator and tend to [use](https://docs.pytest.org/en/latest/parametrize.html) it a [lot](https://github.com/ethpm/ethpm-cli/blob/master/tests/core/test_install.py#L29). It lets you test a lot of varied assumptions without a ton of bloat. I'd load it up with any useful or weird edge cases you want to test.

Reply:
To follow up the rest of the review would we want to even test that this function is throwing an error here anymore? Instead make it something like `test_resolve_uri_or_alias_raises_error` and try to hit all the cases

Reply:
Yeah that's a better idea, i'd just write tests that `resolve_uri_or_alias` fails/passes for all the various edge cases.